websockets/bridge.go

```diff
@@ -0,0 +1,116 @@
+package websockets
+
+import (
+	"fmt"
+
+	"github.com/gorilla/websocket"
+	"github.com/pokt-network/poktroll/pkg/polylog"
+)
+
+// bridge routes data between an Endpoint and a Client. One bridge represents
+// a single WebSocket connection between a Client and a WebSocket Endpoint.
+//
+// Full data flow: Client <------> PATH <------> WebSocket Endpoint
+type bridge struct {
```

Comment:
Do you mind adding a a few comments to `endpointConn`, `clientConn`, etc...

It's very clean, but I have to think a lot and read the code to understand where/how they're being used.

Why are msgCHan and stopChan shared between the two?

Etc...